guidelines/sc/20/visual-presentation.html

```diff
@@ -13,5 +13,7 @@ <h4>Visual Presentation</h4>
      <li>Line spacing (leading) is at least space-and-a-half within paragraphs, and paragraph spacing is at least 1.5 times larger than the line spacing.</li>   
      <li>Text can be resized without assistive technology up to 200 percent in a way that does not require the user to scroll horizontally to read a line of text <a>on a full-screen window</a>.</li>   
   </ul>
+
+<p class="note">The 'mechanism' for achieving the presentation can be provided by the user-agent, which can manipulate the content on behalf of the user. It is not a requirement for the author to provide the mechanism. Some writing systems have additional or different requirements, or do not customarily use features such as paragraph spacing. When a specific feature is inapplicable to a writing system, this criterion does not require a mechanism to achieve that feature for content authored in that writing system.</p>
```

Comment:
Does this note need to be different from the text suggested in #3347? Can that text be adopted here as well?

Reply:
I thought the confusion on this one was about the 'mechanism', which wasn't understood?

Reply:
The WG felt that the other notes were probably applicable here as well.

I agree that there was confusion about the 'mechanism' and this might be suitable for the purpose. I'll propose to our group approving this paragraph.

Reply:
Hi @aphillips, I think the language of the SC is different enough to justify a different note underneath. It's less about the values and more about the 'mechanism'. I'd like to merge this so we can publish...

Reply:
I spent some time working through 1.4.8 Visual Presentation (AAA) and 1.4.12 Text Spacing (AA). I didn't figure out how to put my thoughts in GitHub. For now, they are in a Google Doc:
**[WCAG 2 wording for 1.4.8 and 1.4.12 to better address internationalization (i18n)](https://docs.google.com/document/d/1BoPr_AXLoSTAvcm6OOGVm-2FmbIKC7rKTifhnxucQ6Q/edit)**
Apologies if that is more complication for others.

Please feel free to comment in that Google doc, or move the info to GitHub, as is best for all.

(fyi @iadawn)

Reply:
On the face of the spec I think we need to use a correct term. If there isn't a more understandable term that is also correct, then we explain the term in the understanding doc.
"Writing system" at least has a googlable wikipedia page.

Reply:
Note that it is also a term in the I18N glossary, and, as such, you can link it directly from your spec. I18N encourages linking our glossary as a termref source to keep from having to define our jargon yourself. (In this case, the writing system definition quotes Unicode's glossary...)

Reply:
```suggestion
<p class="note">The 'mechanism' for achieving the presentation can be provided by the user-agent, which can manipulate the content on behalf of the user. It is not a requirement for the author to provide the mechanism. Some [writing systems](https://www.w3.org/TR/i18n-glossary/#dfn-writing-system) have additional or different requirements, or do not customarily use features such as paragraph spacing. When a specific feature is inapplicable to a writing system, this criterion does not require a mechanism to achieve that feature for content authored in that writing system.</p>
```